src/ggplotnim/ggplot_vega.nim

```diff
@@ -292,3 +343,58 @@ proc `+`*(p: GgPlot, d: VegaDraw) =
   let outname = writeVegaFile(plt, d)
   if d.show:
     showVega(outname, d)
+
+proc toVegaHtml*(
+  p: GgPlot,
+  width = 640.0, height = 480.0,
+  pretty = false,
+  divName = "div",
```

Comment:
should this be `divId = "vis"`? (will need to change also in bodyTmpl). Although to be honest `divId` is an ugly name and probably `divName` is better, but I would say "vis" instead of "div" as default values.

Reply:
Yeah, I should change the default name. `divId` vs `divName` I don't have a preference.

Reply:
I mean divId is what it is. the id of the div...